runtime/ops/os.rs

```diff
@@ -105,12 +103,12 @@ fn op_delete_env(state: &mut OpState, key: String) -> Result<(), AnyError> {
 
 #[op]
 fn op_set_exit_code(state: &mut OpState, code: i32) {
-  state.borrow_mut::<Arc<AtomicI32>>().store(code, Relaxed);
+  state.borrow_mut::<ExitCode>().0.store(code, Relaxed);
```

Comment:
The `Relaxed` could be buried in some implementation methods of `ExitCode` rather than being spread around in multiple files.

- `state.borrow_mut::<ExitCode>().set(code);`
- `let exit_code = state.borrow_mut::<ExitCode>().get();`